### DISCUSSION TITLE
Версия 2.1.1

### DISCUSSION THREAD

**Discussion by lirix360** [lirix360/ReadmangaGrabber#60] [Announcements]
Изменения:

- Обновлен адрес сайта ReadManga.

Сообщать об ошибках и предложениях [можно в обсуждениях](https://github.com/lirix360/ReadmangaGrabber/discussions).

<hr /><em>This discussion was created from the release <a href='https://github.com/lirix360/ReadmangaGrabber/releases/tag/v2.1.1'>Версия 2.1.1</a>.</em>

**Comment by Louis-bid** [comment]
При скачивании через осла не работает создание пдф ни по главах ни по томам. Проверяю сейчас в Chrome, позже отпишу.

**Reply by Louis-bid** [reply]
Через Хром все работет.

**Reply by lirix360** [reply]
IE не поддерживается.
